qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/model_fitting/model_fitting_data_selector_view.py

```diff
@@ -98,6 +100,22 @@ def set_selected_y_parameter(self, y_parameter: str) -> int:
         self.y_selector.blockSignals(False)
         return new_index
 
+    def _get_x_parameter_for_update(self, old_x_parameter: str, x_parameters: list, x_parameter_types: list) -> str:
+        if self.y_selector.findText(old_x_parameter) != -1:
+            return old_x_parameter
+        elif 1 in x_parameter_types:
```

Comment:
why 1 and 2?

Reply:
Column types in TableWorkspaces are enumerated so that X is 1, Y is 2. 
https://github.com/mantidproject/mantid/blob/c5e1ddb49cb0f49e8a4324af36bba9f266dbeb63/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/results_tab_widget/results_tab_model.py#L24

Reply:
could we just put `x_col=1, y_col=2` at the top of the file and use them?